### ISSUE TITLE
[Content styleguide] Content doesn’t align with style guidance in VA.gov content style guide. (05.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30020]
#### VFS team name
eBenefits

#### VFS product name
Dependency Verification

#### Point of Contact/Reviewers
Peggy Gannon (Content)

---

### Platform Issue
Content doesn’t align with style guidance in VA.gov content style guide.

### Issue Details
H1 should sentence-cased

### VA.gov Experience Standard
[Category Number 05, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
"Your VA Dependents" should be in sentence-case:
Your VA dependents

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved
